AnnService/src/Core/CommonDataStructure.cpp

```diff
@@ -3,130 +3,102 @@
 
 #include "inc/Core/CommonDataStructure.h"
 
-using namespace SPTAG;
+namespace SPTAG
+{ 
 
 const ByteArray ByteArray::c_empty;
 
-ByteArray::ByteArray()
-    : m_data(nullptr),
-    m_length(0)
+ByteArray::ByteArray() noexcept
+    : m_data(nullptr)
+    , m_length(0)
```

Comment:
please keep original formatting.

Reply:
I have reverted the changes done in member initializer lists